protocol/communities/manager.go

```diff
@@ -1543,14 +1548,11 @@ func (m *Manager) HandleCommunityRequestToJoin(signer *ecdsa.PublicKey, request
 			}
 		}
 
+		accountsAndChainIDs := revealedAccountsToAccountsAndChainIDsCombination(request.RevealedAccounts)
+
 		// provided wallet addresses seem to be legit, so let's check
 		// if the necessary token permission funds exist
-		verifiedAddresses := make([]gethcommon.Address, 0)
-		for walletAddress := range request.RevealedAddresses {
-			verifiedAddresses = append(verifiedAddresses, gethcommon.HexToAddress(walletAddress))
-		}
-
-		permissionResponse, err := m.checkPermissionToJoin(becomeMemberPermissions, verifiedAddresses, true)
+		permissionResponse, err := m.checkPermissionToJoin(becomeMemberPermissions, accountsAndChainIDs, true)
```

Comment:
@cammellos  Here's another usage where the accounts/chainIDs combination is received via the request-to-join